src/components/TodoList/TodoList.tsx

```diff
@@ -25,198 +36,54 @@ export const TodoList: React.FC = () => {
-            <td className="is-vcentered">9</td>
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">molestiae perspiciatis ipsa</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">10</td>
-
-            <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
-            </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">
-                illo est ratione doloremque quia maiores aut
-              </p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
+          {todos.map(todo => (
+            <tr
+              key={todo.id}
+              data-cy="todo"
+              className={cn({
+                'has-background-info-light': todo.id === openedTodo?.id,
+              })}
+            >
+              <td className="is-vcentered">{todo.id}</td>
+              <td className="is-vcentered">
+                {todo.completed && (
+                  <span className="icon" data-cy="iconCompleted">
+                    <i className="fas fa-check" />
+                  </span>
+                )}
+              </td>
+
+              <td className="is-vcentered is-expanded">
+                <p
+                  className={cn({
+                    'has-text-danger': !todo.completed,
+                    'has-text-success': todo.completed,
+                  })}
+                >
+                  {todo.title}
+                </p>
+              </td>
+
+              <td className="has-text-right is-vcentered">
+                <button
+                  data-cy="selectButton"
+                  className="button"
+                  type="button"
+                  onClick={() => handleOpenTodoModal(todo)}
+                >
+                  <span className="icon">
+                    <i
+                      className={cn('far', {
+                        'fa-eye': openedTodo?.id !== todo.id,
+                        'fa-eye-slash': openedTodo?.id === todo.id,
+                      })}
+                    />
+                  </span>
+                </button>
+              </td>
+              
+            </tr>
+          ))}
```

Comment:
Consider adding a check to ensure `todos` is not empty before mapping over it. This can prevent unnecessary rendering of an empty table body and improve performance.